ScanCommand/src/test/java/io/ballerina/scan/Main.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.scan;
+
+import java.io.PrintStream;
+
+public class Main {
+
+    private static final PrintStream outputStream = System.out;
+
+    public static void main(String[] args) {
+
+        outputStream.println("Class used for testing scan tool features");
+    }
+}
+
```

Comment:
Extra new line in here

Reply:
Fixed in [f569dee](https://github.com/ballerina-platform/static-code-analysis-tool/pull/2/commits/f569deea8e438e558f4505d64002edaa768de6f0)